native/SampleApps/RestAPIExplorer/RestAPIExplorer/Classes/AppDelegate.m

```diff
@@ -235,7 +222,7 @@ - (void)handleSdkManagerLogout
             if ([allAccounts count] == 1) {
                 [SFUserAccountManager sharedInstance].currentUser = ([SFUserAccountManager sharedInstance].allUserAccounts)[0];
             }
-            [[SalesforceSDKManager sharedManager] launch];
+            [self startAuthentication:nil];
```

Comment:
These needed to be amended to explicitly restart the auth process, since `launch` doesn't handle it in this case.